app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
     # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
     # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = []
+        for car in cars:
+            if car.clean_mark <= self.clean_power:
+                income.append(self.calculate_washing_price(car))
+                self.wash_single_car(car)
+        return round(sum(income), 1)
+
```

Comment:
avoid the need to create a separate list `income` to store individual incomes and simplifies the code by performing the calculations directly within the loop.